ush/config_defaults.sh

```diff
@@ -1857,3 +1860,9 @@ CLEAN_OLDRUN_HRS="48"
 CLEAN_OLDFCST_HRS="24"
 CLEAN_OLDSTMPPOST_HRS="24"
 CLEAN_NWGES_HRS="72"
+#
+#-----------------------------------------------------------------------
+#
+# Add option to run gsi for obs verif
+RUN_GSI="FALSE"
```

Comment:
Good point! Maybe change it to RUN_OBSERVER?

Reply:
So, you run this GSI observer to collect OmB/OmA information for verification? I am curious how you will use it?

For the variable name, suggest changing to RUN_GSI_OBSERVER. But this still has a minor issue, when the ensDA workflow is mature, will you run GSI observer to prepare OmB for ensemble DA? 

Also, I did not find any use of this variable in your workflow. Is it still under development?

Reply:
Yes, this GSI observer is used to collect OMB information. And this will be used for EnKF later, i.e., prepare OmB for ensemble DA. 
Renamed "RUN_GSI" to be "DO_GSIOBSERVER" to make it more readable, also use "DO_" to be consistent with other settings. 
Added the section to run GSI observer in the FV3LAM_wflow.xml, and also related scripts.

Reply:
Great! Thanks!